internal/boot/aws.go

```diff
@@ -151,17 +151,17 @@ func DescribeEC2Image(e *ec2.EC2, imageName string) (*imageDescription, error) {
 	}, nil
```

Comment:
Commit message says "not a functional change", but it kinda is, isn't it? But I get the point, and the intention stays the same, so... ooook :)

Reply:
:) you are right